app/views/publishers/home.html.slim

```diff
@@ -38,7 +38,7 @@ script id="choose-channel-type" type="text/html"
             #promo-referring-fans= t("promo.activate.for_referring_your_fans")
           .promo-panel-item.promo-panel-item-button
             = form_for current_publisher, { method: :get, url: promo_registrations_path } do |f|
-              = f.submit(t("promo.activate.button-dashboard").upcase, class: "btn btn-primary", id: "activate-promo-dashboard-button" )
+              = f.submit(t("promo.activate.button-dashboard").upcase, class: "btn btn-primary", id: "activate-promo-dashboard-button", :"data-piwik-action" => "ActivatePromoClicked", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "PromoFlow")
```

Comment:
@alexwykoff this is the "learn more" button displayed in the promo panel on the dashboard.  clicking this button will bring you to the activate page where you can activate, but will not activate the promo by itself.  yes, the css class name is misleading.  i would would remove this change or give it a distinct piwik value from the promo activate button.